meson.build

```diff
@@ -0,0 +1,471 @@
+project('naev', 'c',
+   version : '0.8.0',
+   default_options : [
+      'warning_level=1',
+      'optimization=g',
+      'c_std=gnu11',
+      'werror=false'
+   ],
+   meson_version: '>=0.55.0')
+
+
+####
+# Source lists
+####
+source = files(
```

Comment:
Would it be better if some of this is in a `src/build.meson` file? That's how the autotools configuration is currently set up. Looks like the same convention is at least sometimes used with Meson: https://gitlab.gnome.org/GNOME/mutter/-/blob/master/src/meson.build

Reply:
Maybe? It's largely a question of how we want to organize it I think. Meson organizes the build results according to the structure of the meson.build files. At present, the naev binary is placed into the root of the source tree, so I put all the binary stuff at the root so that Meson would put the naev binary into the root of the build directory. I could move some of the declarations into src/meson.build without affecting that for cleanliness.